libraries/helpers.rb

```diff
@@ -190,7 +190,7 @@ def dhcpd_systemd_unit_content(ip_version, config_file)
         dhcp6 = ip_version.eql?(:ipv6)
 
         case node['platform']
-        when 'amazon', 'centos', 'fedora', 'rhel'
+        when 'amazon', 'centos', 'fedora', 'redhat'
```

Comment:
This should potentially get converted to a platform_family check depending on what's below line 196

Reply:
Unless there's a way to distinguish between Debian and Ubuntu in a platform_family then it's going to have to stay as a family check. I agree having to specify all the RH based platforms looks untidy.